<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>center3</class>
 <widget class="QWidget" name="center3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>50</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>120</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>120</width>
    <height>50</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string>行動是治癒恐懼的良藥，而猶豫、拖延將不斷滋養恐懼</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#center3{
background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <widget class="QLabel" name="memUse">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
font: 11pt &quot;Microsoft YaHei UI&quot;;
color: rgb(255, 255, 255);
border-radius: 25px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>7</y>
     <width>118</width>
     <height>36</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>36</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>118</width>
     <height>36</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#frame{
background-color: #FCFCFC;
border: 2px solid gray;
border-radius: 18px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>44</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>13</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(152, 79, 78);
font: 7pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>↑</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="netUp">
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;Microsoft YaHei UI&quot;;
color: #E09A13;
font-weight: bold;</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>10</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(45, 130, 85);
font: 7pt &quot;黑体&quot;;</string>
      </property>
      <property name="text">
       <string>↓</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="netDown">
      <property name="styleSheet">
       <string notr="true">font: 7pt &quot;Microsoft YaHei UI&quot;;
color: #7CE00E;
font-weight: bold;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="memColorFrame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 255, 0, 255), stop:0.87 rgba(0, 100, 0, 255), stop:0.90 #eeeeee, stop:0.99 #ffffff, stop:1 #ffffff);
border-radius: 24px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>frame_2</zorder>
  <zorder>memColorFrame</zorder>
  <zorder>memUse</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
